src/entities/expenses/ui/ExpensesList/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { type FC } from 'react';
+import { useExpensesQuery } from '@entities/expenses/model';
+import { Expense } from '../Expense';
+
+export const ExpensesList: FC = () => {
+  const { isLoading, isError, data, error } = useExpensesQuery();
+
+  if (isError) {
+    return <div>{error?.message}</div>;
+  }
+
+  return (
+    <>
+      {isLoading ? (
+        <div>Loading...</div>
+      ) : (
+        <ul className='w-full max-w-[343px] tablet:max-w-[966px]'>
+          {data?.expenses.map((expense, index) => (
+            <Expense
+              key={expense._id}
+              bg={index % 2 === 0 ? 'bg-[#262626]' : 'bg-inherit'}
```

Comment:
It should be done with help tailwind css, not js
https://cruip.com/tailwind-css-nth-child-selector-cheat-sheet/#nth-selector-04